src/controllers/v1/credentials/CredentialController.ts

```diff
@@ -300,4 +306,27 @@ export class CredentialController extends Controller {
       throw error
     }
   }
+
+  /**
+   * Send problem report regarding a credential
+   * to the connection associated with the credential exchange record.
+   *
+   * @param options
+   * @returns CredentialExchangeRecord
+   */
+  @Example<SendCredentialProblemReportOptions>(SendCredentialProblemReportOptionsExample)
+  @Post('/send-problem-report')
```

Comment:
This should be on something like `/:credentialRecordId/send-problem-report` and then have the credential record Id passed as a path parameter. That probably updates the`SendCredentialProblemReportOptions` being passed in aswell